src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -367,20 +389,16 @@ private void determineTerritoriesToAttack(final Map<Territory, ProAttackTerritor
         ProLogger.trace(patd.getResultString() + " with attackers: " + patd.getUnits());
         final double estimate =
             battleUtils.estimateStrengthDifference(t, patd.getUnits(), patd.getMaxEnemyDefenders(player, data));
-        final ProBattleResultData result = patd.getBattleResult();
-        double winPercentage = WIN_PERCENTAGE;
-        if (patd.isCanAttack() || territoriesToTryToAttack.size() == 1) {
-          winPercentage = MIN_WIN_PERCENTAGE;
-        }
-        if (estimate < patd.getStrengthEstimate()
-            && (result.getWinPercentage() < winPercentage || !result.isHasLandUnitRemaining())) {
+        final ProBattleResult result = patd.getBattleResult();
+        if (!patd.isStrafing() && estimate < patd.getStrengthEstimate()
```

Comment:
complicated boolean expression. Some suggestions, simplify by moving to a method so it can then be abstracted away, break up into intermediary expressions that can then label the parts of the expression and what they represent.